src/ontology/uberon-edit.obo

```diff
@@ -224371,6 +224373,19 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
 relationship: part_of UBERON:0001114 ! right lobe of liver
 property_value: dcterms-date "2023-05-15T10:52:39Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600017
+name: bronchopulmonary segment
+def: "A discrete anatomical unit of the lung that is composed of a segmental bronchus and the lung tissue supplied by that bronchus. A bronchopulmonary segment exhibits a pyramidal shape, with its apex pointing towards the center of the lung and its base toward the pleural surface." [doi:10.1016/j.mpaic.2008.09.022, FMA:7312, PMID:18271171, PMID:33620799]
+comment: A bronchopulmonary segment is surrounded by connective tissue septas, which are continuous with the pleural surface and prevent air collateralization between segments somewhat. In humans, the right lung has ten bronchopulmonary segments, with eight or nine bronchopulmonary segments in the left lung. {xref="doi:10.1016/j.mpaic.2008.09.022", xref="PMID:18271171"}
```

Comment:
Thank you for your comment. The reason I have added that sentence in the comment is because it does not define the bronchopulmonary segment, but describes structures that surround it.
I will make the edits you propose, thanks!

Reply:
> is because it does not define the bronchopulmonary segment, but describes structures that surround it.

My understanding is that it defines the boundary, but will defer to requester/SME.